packages-ts/gauntlet-terra-contracts/networks/.env.testnet-bombay

```diff
@@ -1,6 +1,8 @@
 NODE_URL=https://bombay-lcd.terra.dev
 CHAIN_ID=bombay-12
 DEFAULT_GAS_PRICE=0.5
+RDD=../reference-data-directory/directory-terra-testnet-bombay.json
```

Comment:
The usual setup would be to run repositories side by side, so please tweak this default to expect `https://github.com/smartcontractkit/chainlink-terra` & `reference-data-directory` side by side.

Reply:
The relative path if given here corresponding to the directory from where the gauntlet command is run (the root of terra repository), not to the location of the env file.